pkg/runs/runsGet.go

```diff
@@ -203,3 +229,66 @@ func GetRunsFromRestApi(
 
 	return results, err
 }
+
+func getTimesFromAge(age string) (int, int, error) {
```

Comment:
Don't see any unit tests for this bit ? would be easier to unit test it in isolation.